code/modules/mob/living/carbon/human/species/plasmaman.dm

```diff
@@ -2,7 +2,6 @@
 	name = "Plasmaman"
 	name_plural = "Plasmamen"
 	icobase = 'icons/mob/human_races/r_plasmaman_sb.dmi'
-	deform = 'icons/mob/human_races/r_plasmaman_pb.dmi'  // TODO: Need deform.
```

Comment:
This is the only one I'm keeping; we need to standardize on one or the other.

Quite frankly, the "deformed" (it's not) one is better, but I'll save that for another PR.